tests/security/bug10453.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Bug #10453 (using a high amount of filters for nefarious purposes)
+--FILE--
+<?php
+$fp = fopen('php://output', 'w');
+for($i=0; $i<10; $i++)
+    stream_filter_append($fp, 'string.rot13');
```

Comment:
Should a testcase be added that does not use `stream_filter_append()`, but puts the full filter chain into the filename directly?